rfcs/convergence/changelog-guidelines.md

```diff
@@ -0,0 +1,130 @@
+
+### Changelog entries that are focused on the PR
+
+Quite often, a PR will address/fix an issue that spans several packages. While it's good practice to keep PRs as
+small as possible, it is still possible that mutliple changes in different packages that need to be modified. This
+can also be acceptable when an internal API is modified in a non breaking way, but several consumers need to be updated
+at the same time.
+
+It is very easy to write a single changefile message during PR creating and apply it to all the changed packages in
+the PR. This makes the change harder to understand since the contents might only apply vaguely, or not at all to the package.
+
+A simple example would be `Fix dependencies`, this kind of generic description can be applied to a PR when linked to
+an issue, but has an incredibly vague meaning with applied to the changelog of a package.
+
+## Detailed design or proposal
+
+### Semantic labels for changefile entries
+
+Each changefile entry should have a message that has a semantic label that is inspired by [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/).
+This RFC proposes four labels in the following form:
+
+```
+feat:  This label communicates that a feature was added to the package
+fix:   This label communicates that a fix was applied to the package
+chore: This label communicates that changes which should not have an affect on th user were applied
+docs:  This label communicates that documentation was changed
+```
+
+We can configure [danger.js](https://danger.systems/js/) to validate changefile entries in the PR that a correct
+label has been applied.
+
+### Stricter reviews for changefile comments
+
+There is no easy way to automate the quality of a changelog entry, this has to be done by humans in the PR review.
+This proposal tries to spread awareness that changefile entries are directly a part of our changelog and they should
+be reviewed strictly.
+
+### Stricter review of `type` and `dependentChangeType`
+
+Here is a sample beachball changefile:
+
+```json
+{
+  "type": "patch",
+  "comment": "This comment is the message used in the changelog entry",
+  "packageName": "@fluentui/babel-make-styles",
+  "email": "jane.doe@contoso.com",
+  "dependentChangeType": "patch"
+}
+```
+
+The value of `type` determines what kind of version bump will be applied to the package on release. It can take the
+following values:
+
+- none
+- prerelease
+- patch
+- minor
+- major
+
+These values are identical to standard semver bumps, and will result in bumps of that type being applied to the package
+on release. Beachball will always pick the highest bump type among all changefiles for a package when bumping. A value
+of `none` will result in no bumps applied to the package. The `comment` will not be applied to the changelog when `none`
+is used for `type`.
+
+The value of `dependentChangeType` is the same as `type`. However, this bump affects upstream packages that depend on the
```

Comment:
nit, should probably be something like "The possible values of `dependentChangeType` are the same as for `type`" (right now it could be read as they should literally be the same in the change file, which is often not true)